crypto/src/mls/conversation/leaf_node_validation.rs

```diff
@@ -149,32 +138,29 @@ mod tests {
         #[apply(all_cred_cipher)]
         #[wasm_bindgen_test]
         async fn should_validate_leaf_node_when_receiving_add_commit(case: TestContext) {
-            let [alice_central, bob_central, charlie_central] = case.sessions().await;
+            #[expect(unused_variables)]
+            let [alice, bob, charlie] = case.sessions().await;
+            // Skipping, see reason below
+            return;
+            #[expect(unreachable_code)]
```

Comment:
Similarly here. Also this is a great example of "testing the library". In fact, would say this whole module should be removed.